src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -3102,11 +3125,39 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
                 llvm_symtab_fn_names[fn_name] = h;
                 F = llvm::Function::Create(function_type,
                     llvm::Function::ExternalLinkage, fn_name, module.get());
+
+                // Add Debugging information to the LLVM function F
+                llvm::DIFile *Unit = DBuilder->createFile(
+                    debug_CU->getFilename(),
+                    debug_CU->getDirectory());
+                llvm::DIScope *FContext = Unit;
+                unsigned LineNo = 0;
+                unsigned ScopeLine = 0;
+                std::string fn_debug_name = x.m_name;
+                llvm::SmallVector<llvm::Metadata *, 8> el_types;
+                llvm::DIType *DblTy = DBuilder->createBasicType("double", 64,
+                    llvm::dwarf::DW_ATE_float);
+                el_types.push_back(DblTy);
+                el_types.push_back(DblTy);
+                el_types.push_back(DblTy);
```

Comment:
I think we don't even care about this for stacktraces, so we can just use empty `el_types` (if it works).

Reply:
Yup, I felt the same. So, replacing it with 0 (null)
```
llvm::DISubroutineType *dtype = DBuilder->createSubroutineType(
                    DBuilder->getOrCreateTypeArray(0));
```